app/controllers/businesspartners_controller.rb

```diff
@@ -0,0 +1,36 @@
+class BusinesspartnersController < ApplicationController
+
+  def receive_data
+    @basic_info = params[:basicInfo]
+    @contact_info = params[:contactInfo]
+
+    @business_partner = BusinessPartner.new
+    @contact_card = ContactCard.new
+
+    @business_partner.contact_card = @contact_card
+    @business_partner.from_json(@basic_info)
+    @contact_card.from_json(@contact_info)
+
+    @contact_card.save!
+    @business_partner.save!
```

Comment:
A place for a transaction. If `@contact_card` saving succeds and `@business_partner` doesn't, nothing will be saved into database.